http-ratelimiting/src/request.rs

```diff
@@ -180,6 +180,8 @@ pub enum Path {
     GuildsIdChannels(u64),
     /// Operating on one of the user's guilds' widget.
     GuildsIdWidget(u64),
```

Comment:
i'd like to have the names be more appropriate (`guilds/id/widget.json` -> `GuildsIdWidgetJson`, and `guilds/id/widget` -> `GuildsIdWidget`) but that would have to take place on `next`